source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3174,6 +3174,26 @@ Note that :mixxx:coref:`[Library],MoveUp` and other Move and Scroll controls emu
    .. versionadded:: 2.1.0
 
 
+.. mixxx:control:: [Library],MoveTrackUp
+
+   Equivalent to pressing the :kbd:`Alt` + :kbd:`Up` key on the keyboard
+
+   :range: Binary
+   :feedback: Move selected track up
+
+   .. versionadded:: 2.5.0
+
+
+.. mixxx:control:: [Library],MoveTrackDown
+
+   Equivalent to pressing the :kbd:`Alt` + :kbd:`Down` key on the keyboard
+
+   :range: Binary
+   :feedback: Move selected track down
+
+   .. versionadded:: 2.5.0
+
+
```

Comment:
Note that the controls seem to be ordered alphabetically, but actually it's better to have them grouped by 'function'/effect. Please move the block down so `MoveVertical` is below `MoveUp` again.

Reply:
Oh sure, I didn''t realize it was related